README.md

```diff
@@ -35,29 +58,74 @@ Anyone can run your frontend with `garn run frontend`, backend with `garn run
 backend`, or both with `garn run startAll`. All without needing to worry about
 having the correct version of `go`, `nodejs`, or anything else installed.
 
-garn is powered by [Nix](https://nixos.org/), so you get portable and
-reproducible builds and development environments -- all this while running
-natively on your developer machines without the hassle and performance penalty
-of using e.g. docker.
+You may have noticed the use of import URLs in the file above. This is supported
+by Deno, which we use under the hood. But you don't need to install that either.
+In fact, **garn** might just be one of the last things you have to manually
+install.[^2]
+
+**garn** is powered by [Nix](https://nixos.org/), so you get portable and
+reproducible builds and development environments.
+
 
+---
+
+## Table of contents
+
+- [Getting Started](#getting-started)
+- [Commands](#commands)
+- [Core Concepts](#core-concepts)
+- [Languages and stacks](#languages-and-stacks)
+- [How it works](#how-it-works)
+- [Comparison to other tools](#comparison-to-other-tools)
+- [Typescript API](#typescript-api)
+
+-
 ## Getting started
 
-### Install garn
+1) **Install garn**:
 
-```bash
-sh <(curl --proto '=https' --tlsv1.2 -sSf https://garn.io/install.sh)
-```
+   ```bash
+   sh <(curl --proto '=https' --tlsv1.2 -sSf https://garn.io/install.sh)
+   ```
+
+   `garn` needs [`nix`](https://nixos.org/) to be installed, so if you don't
+   have nix already the above installer will install nix first, after asking
+   for confirmation.
+
+   <details>
```

Comment:
You're using html tags here, which works, but it breaks markdown inside the tags apparently. But you're using that in various places. So e.g. `**garn**` or `[this page](https://...)` doesn't work as expected.